language-design.md

```diff
@@ -62,7 +175,50 @@ the commenters?
 
 **Response**
 
-
+Pavlus is arguing that programming languages should be designed to be easier
+for the average person to understand and learn. While we agree with this
+general argument of the article, the actual specifics Pavlus uses to bolster
+his claim are not convincing. The commenters point out that Pavlus is
+emphasizing the difficulty of learning a language for those new to the
+language, but they argue that a programming language should be designed more
+with its users in mind, in that ease of use should be the concern, not ease
+of learning.
+
+We mostly agree with the commenters. The language Pavlus seems to be after is a
+language for teaching programming, not one for actually programming. The reason
```

Comment:
cool interpretation, hadn't considered this idea either. can you think of any languages that are already "teaching languages"? as in languages that are almost exclusively used for teaching how to program?